### ISSUE TITLE
Mathjs in Angular CLI Webworker

### ISSUE THREAD

**Issue by dominikks** [josdejong/mathjs#1867]
Hello,

I am currently running into some problems when trying to use mathjs in a webworker in an Angular CLI project (Angular CLI version is 9.1.7). 

When I import any mathjs function into the worker, I get the following error in the browser console. If I comment out the mathjs import, the worker runs without a problem.
```
index.js:7 Uncaught ReferenceError: window is not defined
    at Object../node_modules/seed-random/index.js (index.js:7)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/mathjs/es/function/probability/util/seededRNG.js (seededRNG.js:1)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/mathjs/es/function/probability/pickRandom.js (pickRandom.js:1)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/mathjs/es/factoriesAny.js (factoriesAny.js:1)
    at __webpack_require__ (bootstrap:19)
    at Module../node_modules/mathjs/es/entry/pureFunctionsAny.generated.js (pureFunctionsAny.generated.js:1)
    at __webpack_require__ (bootstrap:19)
```

The referenced line is
```
var GLOBAL = typeof global === 'undefined' ? window : global;
```
and is part of the [seed-random](https://github.com/ForbesLindesay/seed-random/blob/04f50d84870872bf38ea6283b1846202acf9ef44/index.js#L7) library used in mathjs.

I created a GitHub Repo as a minimal working example [here](https://github.com/dominikks/angular-cli-webworker-mathjs-error). You can run it by calling
```
npm install
npm start -- --open
```
and should see the error in the browser console as soon as the website opens.

As there is an example of mathjs running in a webworker on the official page ([here](https://mathjs.org/examples/browser/webworkers/index.html)), I assume it should be also be possible in an Angular project. However I am unsure of how to get it to work.

I don't know whether this problem can be solved in mathjs, but I would appreciate it if you have any advice. Thanks in advance!

**Comment by josdejong** [comment]
Thanks for your detailed report and for working out an example to reproduce this Dominik.

As a workaround, if you replace the way you import:

```js
import { sqrt } from 'mathjs';
```

with:

```js
import { sqrt } from 'mathjs/dist/math.js';
```

It all works. So there is probably something in the way mathjs is bundled that makes the error with undefined window not occur.

**Comment by josdejong** [comment]
Can anyone help debugging this further and creeate a real fix? (That may be a fix in `seed-random` or the way we load seed-random)

**Comment by dominikks** [comment]
You are right, I got it to work by changing the import as you described. Thanks a lot!
